src/fourlang/dep_to_4lang.py

```diff
@@ -244,6 +247,13 @@ def create_from_line(line):
             # logging.info('adding new rel from: {0}'.format(dep))
             rel = dep.split('_', 1)[1].upper()
 
+        # Universal Dependencies
+        if ((dep.startswith('acl:') and not dep.startswith('acl:relcl')) or
+                dep.startswith('advcl:') or
+                dep.startswith('nmod:')) and rel is None:
+            logging.info('adding new rel from: {0}'.format(dep))
+            rel = dep.split(':', 1)[1].upper()
```

Comment:
@Eszti can you maybe show 1-2 examples for this case in the comments? Just so we'll know in the future what this is about